Source/ChocolateyGui.Common.Windows/ViewModels/Items/PackageViewModel.cs

```diff
@@ -148,17 +148,29 @@ public string[] Authors
             set { SetPropertyValue(ref _authors, value); }
         }
 
-        public bool CanInstall => _allowedCommandsService.IsInstallCommandAllowed && !IsInstalled;
+        public bool CanInstall => !IsInstalled;

```

Comment:
The `Can...` properties needs the allowed check too. Without that the buttons/menus are allowed to use.

Reply:
The `CanInstall` boolean is tied to the Visibility property of the button.  So I am not sure if we want to add the AllowedCommands property to this.  Before this PR, I was showing/hiding the buttons based on whether the operation was allowed and if the operation currently made sense.  With this change, the button is now shown/hidden based on whether the operation makes sense (i.e. the install button is shown, if the package isn't currently installed), and then using the new behavior, it will either be enabled/disabled, based on whether the operation is allowed based on the current Chocolatey configuration.

Does this make sense?  Or am I missing something?

Reply:
The new behavior (ToolTipBehavior) is only for the ToolTip. It doesn't handle the visibility or command usage of the element. So now if the `IsInstallAllowed` is set to false, you will see the disabled feature tooltip, but the button is still available and the user can install the package if it's not installed.

```
<Button Padding="10" Margin="5 0" Command="{commands:DataContextCommandAdapter Install}">

</Button>
```